### PR TITLE
Issues with touch device support

### PR DESCRIPTION
**Pull request by maxkfranz**
Ref :  Issues with touch device support #273 

@d2fong Would you please review this?  It affects the help system.  It works as far as I can tell, but it would be good to have a second person try it out.

@jvwong This PR contains the same sort of fixes that should be applied to pc/app-ui.  (Some fixes don't apply to app-ui, but several do.)  We should add this task to one of the student's queue.

### CODE REVIEW

**Comment by d2fong** [comment]
Here is what the help system looks like on chromes mobile view:
The interaction tooltips still get cut off and they dont seem to point to the interaction.

screen shot 2018-05-25 at 1 41 01 pm

I am guessing some of the tooltip changes need to be applied to these tooltips too?

**Comment by d2fong** [comment]
I think it would be useful to apply these changes to the help system like calculating the max/min width for the tooltips

File: src/styles/notification.css [hunk 1]
```diff
@@ -95,10 +95,22 @@
   max-width: calc(100vw - 2em);
 }
 
+@media (min-width: 500px) {
```
Comment (d2fong) [inline src/styles/notification.css]: You use 500px here but in cy/tippy.js you use 650 to determine whether there is a small screen or not.  Should these two numbers be the same or not?

**Comment by d2fong** [comment]
Ok in that case, I didn't notice anything wrong with the help system in this branch.

File: src/styles/editor/editor.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .editor {
-  height: 100vh;
-  width: 100vw;
-  position: relative;
+  height: 100%;
```
Comment (d2fong) [inline src/styles/editor/editor.css]: Why do these need to be changed to percentages?

Comment (maxkfranz) [reply src/styles/editor/editor.css]: I've found that `100vh` isn't what someone would normally expect on iOS -- it includes the bottom toolbar such that anything with `100vh` will overflow the screen.

So then, the only safe way to use `vh` units is for something much smaller than the full screen, like `50vh`.

Comment (d2fong) [reply src/styles/editor/editor.css]: Ok that sounds like something I may have to consider for the help system overlay.  Anyways I don't see anything wrong with the code so I approve of the PR.

**Pull request merged**